pkg/k8shandler/visualization.go

```diff
@@ -45,16 +50,12 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateVisualization() (err
 			oauthSecret = utils.GetRandomWord(64)
 			utils.WriteToWorkingDirFile("kibana-proxy-oauth.secret", oauthSecret)
 		}
-
-		if err = clusterRequest.createOrUpdateKibanaSecret(oauthSecret); err != nil {
```

Comment:
is there a reason you removed this? I would expect that we don't see the secret created at all now...

Reply:
I dont think this was intentional... may be a bad merge or the way I brought in the file